VideoLocker/src/main/java/org/edx/mobile/view/CreateGroupFragment.java

```diff
@@ -165,12 +169,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         addButton = (Button) rootView.findViewById(R.id.next_create_group);
         addButton.setOnClickListener(this);
 
-        View shareAppButton = rootView.findViewById(R.id.btn_share_app);
+        TextView shareAppButton = (TextView)rootView.findViewById(R.id.btn_share_app);
+        CharSequence message = ResourceUtil.getFormattedString(getResources(), R.string.btn_share_app, "platform_name", config.getPlatformName());
+        shareAppButton.setText(message);
         shareAppButton.setOnClickListener(this);
 
         errorLayout = rootView.findViewById(R.id.error_layout);
         progress = rootView.findViewById(R.id.progress);
         formLayout = rootView.findViewById(R.id.create_group_form);
+        CharSequence errorText = ResourceUtil.getFormattedString(getResources(), R.string.error_no_friends_connected, "platform_name", config.getPlatformName());
+        TextView errorMessage = (TextView)rootView.findViewById(R.id.error_message);
```

Comment:
Maybe change this variable name to indicate that this is a widget (`errorMessageView` maybe)?